docs/registries.conf.5.md

```diff
@@ -0,0 +1,28 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
```

Comment:
Is the `[][]` syntax supposed to be a link?

`s/simple list format/a &/`?

Reply:
was copied from another, similar man page.  I have reworked it without the []'s

Reply:
Is this _intended_ to be a link? If so, _some_ markup is probably required.